crates/l2/utils/config/mod.rs

```diff
@@ -34,3 +34,28 @@ pub fn read_env_file() -> Result<(), errors::ConfigError> {
 
     Ok(())
 }
+
+pub fn read_env_as_lines(
+) -> Result<std::io::Lines<std::io::BufReader<std::fs::File>>, errors::ConfigError> {
+    let env_file_name = std::env::var("ENV_FILE").unwrap_or_else(|_| ".env".to_string());
+    let env_file = std::fs::File::open(env_file_name)?;
+    let reader = std::io::BufReader::new(env_file);
+
+    Ok(reader.lines())
+}
+
+pub fn write_env(lines: Vec<String>) -> Result<(), errors::ConfigError> {
+    let env_file_name = std::env::var("ENV_FILE").unwrap_or_else(|_| ".env".to_string());
```

Comment:
Ditto.

Reply:
Yes, the default name for the envfile is `.env` the `envar` can be used to set the non-relative path if necessary.

Reply:
I got this. By the previous comment I meant that we could have a `const DEFAULT_ENV_FILE_NAME: &str = ".env";` and then use `unwrap_or(DEFAULT_ENV_FILE_NAME.to_owned())`